dev/src/Command/AddComponentCommand.php

```diff
@@ -184,21 +212,150 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $composer->updateMainComposer();
         $composer->createComponentComposer($new->displayName, $new->githubRepo);
 
+        if ($input->getOption('run-owlbot')) {
+            $output->write("\n\nBuilding the library using Bazel\n");
+            $output->write($this->bazelBuildLibrary($new->shortName, $googleApisDir));
+            $output->write("\n\nCopying the Bazel output to the Google Cloud PHP directory\n");
+            $output->write($this->copyBazelOutToGoogleCloudPhp($new->componentName, $googleApisDir, $output));
+            $output->write("\n\nOwlbot post processing\n");
+            $output->write($this->postProcess());
+        }
+
         $output->writeln('');
         $output->writeln('');
         $output->writeln('Success!');
 
         return 0;
     }
 
-    private function loadProtoContent(string $proto): string
+    private function loadRawFileContent(string $file, string $type = 'proto'): string
     {
-        if (file_exists($proto)) {
-            return file_get_contents($proto);
+        if (file_exists($file)) {
+            return file_get_contents($file);
         }
-        $protoUrl = 'https://raw.githubusercontent.com/googleapis/googleapis/master/' . $proto;
+        $fileUrlPath = explode('/googleapis/google/', $file)[1];
+        $fileUrl = 'https://raw.githubusercontent.com/googleapis/googleapis/master/google/' . $fileUrlPath;
         $client = new Client();
-        $response = $client->get($protoUrl);
+        $response = $client->get($fileUrl);
         return (string) $response->getBody();
     }
+
+    private function bazelBuildLibrary(string $libraryName, string $googleApisDir): string
+    {
+        $command = ['bazel', '--version'];
+        $output = $this->runCommand($command);
+        // Extract the version number from the output
+        $match = preg_match('/bazel\s+(?P<version>\d+\.\d+\.\d+)/', $output, $matches);
+        if (!$match || $matches['version'] !== self::BAZEL_VERSION) {
+            throw new RuntimeException('Bazel 6.0.0 is not available');
+        }
+        $command = [
+            'bazel',
+            'query',
+            'filter("-(php)$", kind("rule", //google/cloud/' . $libraryName .'/...:*))'
```

Comment:
This needs to change

Reply:
Resolved, this was the root cause of `shoppingCss` issue @bshaffer has faced.

Reply:
But need a clean logic for this though!

Reply:
What issue is that?

Reply:
the `shoppingCss` is not under `google/cloud`, it is under `google/shopping`, so the rule I gave earlier was not correct, now it has changed from `'filter("-(php)$", kind("rule", //google/cloud/' . $libraryName .'/...:*))'` to `'filter("-(php)$", kind("rule", //' . $component . $libraryName .'/...:*))'`